development/components/form/types-reference/text-preview-type.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TextPreviewType
+---
+
+# TextPreviewType
+
+This form type is used to display a text value without providing an interactive input to edit it. It is based on a hidden input so it could be changed programmatically, or be used just to display a data in a form.
```

Comment:
```suggestion
This form type displays a text value without providing an interactive input to edit it. It is based on a hidden input so it could be changed programmatically, or used just to display data in a form.
```